bootstrap/kubeadm/internal/controllers/kubeadmconfig_controller.go

```diff
@@ -63,6 +63,18 @@ const (
 	KubeadmConfigControllerName = "kubeadmconfig-controller"
 )
 
+var (
+	controlPlaneTaint = corev1.Taint{
```

Comment:
nit: let's add some godoc that those are the taints kubeadm is using and we essentially copied them over